package.json

```diff
@@ -36,6 +36,8 @@
     "cy:open": "cypress open",
     "cy:run:test-no-meilisearch": "cypress run --spec '**/*/test-no-meilisearch.cy.js'",
     "cy:run:test-api-key-required": "cypress run --spec '**/*/test-api-key-required.cy.js'",
+    "cy:run:test-api-key-query-param": "cypress run --spec '**/*/test-api-key-query-param.cy.js'",
+    "cy:run:test-no-api-key": "cypress run --spec '**/*/test-no-api-key-required.cy.js'",
     "cy:run": "cypress run --config excludeSpecPattern=['**/*/test-no-meilisearch.cy.js','**/*/test-api-key-required.cy.js']",
```

Comment:
Initially I had those commands for those specific tests:
- `cy:run:test-no-meilisearch` : tests when there is no Meilisearch instance running
- `cy:run:test-api-key-required` : tests that require a Meilisearch instance running + an API key
- `cy:run` : tests that require a Meilisearch instance running (without an API key here for simplicity)

I had those 3 because I needed to run 3 different instances in the CI. 
I don't really get why you added more commands in the `package.json` nor more jobs in the CI, but I'm available to discuss it of you want :)